src/vs/workbench/parts/terminal/electron-browser/terminal.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import * as os from 'os';
+import platform = require('vs/base/common/platform');
+import processes = require('vs/base/node/processes');
+
+const powershellPath = `${ process.env.SystemRoot }/system32/WindowsPowerShell/v1.0/powershell.exe`;
+const isAtLeastWindows10 = platform.isWindows && parseFloat(os.release()) >= 10;
```

Comment:
I'm not sure if it's necessary, since I'm parsing it to a float.  That means 10.2.22222 and 11.4.4444 will still be >= 10 (and 9.5.555 will not).

I'm happy to take the change - let me know what you want to do.